pkg/controller/operators/olm/requirements.go

```diff
@@ -273,16 +273,7 @@ func (a *Operator) permissionStatus(strategyDetailsDeployment *v1alpha1.Strategy
 			if ownerutil.IsOwnedByKind(sa, v1alpha1.ClusterServiceVersionKind) && !ownerutil.IsOwnedBy(sa, csv) {
 				met = false
 				status.Status = v1alpha1.RequirementStatusReasonNotPresent
```

Comment:
Would this make more sense as `v1alpha1.RequirementStatusReasonPresentNotSatisfied` since the SA exists but the condition of use (not owned by another CSV) isn't satisfied?

Reply:
I'm not totally clear on the the intended semantic difference, but your interpretation seems reasonable to me. The tests aren't verifying the expected status per requirement, so I'll go ahead and fix that too.